image_classification/tensorflow/run.sh

```diff
@@ -4,7 +4,13 @@ RANDOM_SEED=$1
 QUALITY=$2
 set -e
 
-export PYTHONPATH=`pwd`:$PYTHONPATH
+# Register the model as a source root
+export PYTHONPATH="$(pwd):${PYTHONPATH}"
+
+# register the compliance dir as a source root
+pushd ../../compliance
```

Comment:
I might have missed something, but just wondering how do you make the compliance directory available to docker container? I thought you might need to either copy it in dockerfile or mount it at run time, but seems I could not find that in the code.

Reply:
Ummm... that is an excellent point. I have been testing directly in shell. However I absolutely will need to adapt the dependencies to work with the docker workflow of the references. Thanks for bringing this up!